Changed _UNKNOWN_ to _MACHINA_

Heistergand opened the pull request:
Changed unicode decorated text `UN̴̖̈KNȮWṈ ` to simple text `__MACHINA__`.

Following the In-Game story, the unknown faction was given the name `__MACHINA__`, shown in scanners and stock intel as `_̶̱̍_MĄ̷̍CHḬNÄ__`  .

According to the discussion in the Telegram IITC Dev channel, it is more important that the user interface is easy to use. Better readability contributes to this.

Heistergand committed:
Changed _UNKNOWN_ to _MACHINA_

modos189 commented:
Please change to plain `__MACHINA__`

That's why:
1. In the developers telegram chat most people like the readable font
2. ~~I think it is more important: that the portal owner and the owner of the resonators have the same text~~ (up: this change does not affect the name of the owner)

xscreach commented:
Imho the only valid reason to make this change is that stock intell now shows the layer as weird machina too (it was "unknown" before, not sure when that changed).

Heistergand committed:
Changed comment to plain text __MACHINA__

Even though or especially because it concerns only a comment here.

Heistergand committed:
Changed MAC team name to plaintext __MACHINA__

According to the discussion in the telegram IITC Dev channel, it is more important that the user interface is easy to use. Better readability contributes to this.
